services/core-web/src/components/Forms/permits/conditions/Conditions.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import { Divider, Icon, Row, Collapse, Button } from "antd";
+import { openModal, closeModal } from "@common/actions/modalActions";
+import CustomPropTypes from "@/customPropTypes";
+import Condition from "@/components/Forms/permits/conditions/Condition";
+import AddCondition from "@/components/Forms/permits/conditions/AddCondition";
+import { getPermitConditionCategoryOptions, getPermitConditionTypeOptions } from "@common/selectors/staticContentSelectors";
+import { getPermitConditions, getDraftPermitAmendmentForNOW } from "@common/selectors/permitSelectors";
+import { fetchPermitConditions, setEditingConditionFlag } from "@common/actionCreators/permitActionCreator";
+import {
+    getNoticeOfWork
+} from "@common/selectors/noticeOfWorkSelectors";
+import { maxBy } from "lodash";
+
+const { Panel } = Collapse;
+
+const propTypes = {
+    isViewMode: PropTypes.bool.isRequired,
+    conditions: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.any)).isRequired,
+    permitConditionCategoryOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
+    permitConditionTypeOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
+    fetchPermitConditions: PropTypes.func.isRequired,
+    noticeOfWork: CustomPropTypes.importedNOWApplication.isRequired,
+    draftPermitAmendment: CustomPropTypes.permit.isRequired,
+    setEditingConditionFlag: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+};
+
```

Comment:
same here. Are you not getting linting errors for these when committing?

Reply:
I am currently getting an exception when the linter runs. I've put looking into it off, but I'll get it linted before EOD